config/routes.rb

```diff
@@ -4,11 +4,13 @@
   }
 
   namespace :api do
-    namespace :v1 do
+    namespace :v1, defaults: { format: :json } do
```

Comment:
Is `defaults: { format: :json }` needed? Since you have `only_api = true`?

Reply:
It's needed, we tried without it and it tries to render an html instead of a json 🤷‍♂️

Reply:
Which request is having this issue? It's ok to leave that there, just wondering why, maybe the issue is with the headers not sending application/json

Reply:
Get topics and create target requests had the issue and needed that fix